Fix/Update README for ATF changes made in 243 (#2630)
smartdevicelink/sdl_atf_test_scripts: README.md

  * Later the SDL sources destination directory is referenced as `<sdl_core>`
* [Setup ATF](https://github.com/smartdevicelink/sdl_atf).
  * Later the ATF sources destination directory is referenced as `<sdl_atf>`
  * Later the ATF build destination directory is referenced as `<atf_build>`
* Clone [sdl_atf_test_scripts](https://github.com/smartdevicelink/sdl_atf)
  * Later the atf test scripts destination directory is referenced as `<sdl_atf_test_scripts>`

  ```bash
  git clone https://github.com/smartdevicelink/sdl_atf_test_scripts <sdl_atf_test_scripts>
  ```

* Create symlinks in the build `bin` directory to certain directories in `<sdl_atf_test_scripts>`:

  ```bash
  cd <atf_build>/bin
  ln -s <sdl_atf_test_scripts>/files
  ln -s <sdl_atf_test_scripts>/test_sets
  ln -s <sdl_atf_test_scripts>/test_scripts
  ln -s <sdl_atf_test_scripts>/user_modules
  ```

* In your `<sdl_atf>/modules/configuration/base_config.lua`
  * Include the path to your local SDL Core binary (ex. `<sdl_build>/bin/`):
  ```lua
  --- Define path to SDL binary
  -- Example: "/home/user/sdl_build/bin"
  config.pathToSDL = "/home/user/sdl_build/bin"
  ```

>   * Include the path to your local SDL Core Source directory (ex. `<sdl_build>/bin/`):

Reviewer: ```suggestion
  * Include the path to your local SDL Core Source directory (ex. `<sdl_core>/`):
```

  ```lua
  --- Define path to SDL source
  -- Example: "/home/user/sdl_core"
  config.pathToSDLSource = "<sdl_core>"
  ```

>   ATF will use this path to derive the path to the MOBILE_API.xml and HMI_API.xml files in the `<sdl_core>` directory.

Reviewer: ```suggestion
  ATF will use the provided path to derive the location of the MOBILE_API.xml and HMI_API.xml files within the directory.
```

>   * Instead of including the path to your local SDL Core Source directory, you can alternatively include the path to your local HMI_API and MOBILE_API directories:

Reviewer: ```suggestion
  * Instead of including the path to your local SDL Core Source directory, you can alternatively include the paths to your local HMI_API and MOBILE_API directly:
```

Author: `directly` may not be the best word to use here since `pathToMobileInterface`/`pathToHMIInterface` still refers to the directory which contains the API file

```lua
--- Define path to SDL MOBILE interface
  -- Example: "/home/user/sdl_core/tools/rpc_spec"
...
--- Define path to SDL HMI interface
  -- Example: "/home/user/sdl_core/src/components/interfaces"
...
```
Hopefully the changes in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2630/commits/e88412143b08ecb7e64d2e85fe7ff882b27a11ad work better

  ```lua
  --- Define path to SDL MOBILE interface
  -- Example: "/home/user/sdl_core/tools/rpc_spec"
  config.pathToSDLMobileInterface = "/home/user/sdl_core/tools/rpc_spec"
  --- Define path to SDL HMI interface
  -- Example: "/home/user/sdl_core/src/components/interfaces"
  config.pathToSDLHMIInterface = "/home/user/sdl_core/src/components/interfaces"
  ```

>   **NOTE:** If both `pathToSDLSource` and `pathToSDLMobileInterface`/`pathToSDLHMIInterface` are defined in the config file, the `pathToSDLMobileInterface`/`pathToSDLHMIInterface` will be used instead of the path derived from `pathToSDLSource`.

Reviewer: ```suggestion
  **NOTE:** If both `pathToSDLSource` and `pathToSDLMobileInterface`/`pathToSDLHMIInterface` are defined in the config file, the paths provided in `pathToSDLMobileInterface`/`pathToSDLHMIInterface` will override the paths derived from `pathToSDLSource`.
```
